tests/gtest/datahandle_attribute_test.cpp

```diff
@@ -233,4 +277,59 @@ TEST_F(DatahandleAttributeTest, Attribute_Different_Size_Double) {
     delete subvolume;
 }
 
+TEST_F(DatahandleAttributeTest, Attribute_Double_Data_Outside_Intersection_Horizontal_Plane) {
+    DoubleDataHandle& datahandle = double_datahandle;
+    int ilines_before = 3;
+    int ilines_after = 1;
+    int xlines_before = 1;
+    int xlines_after = 2;
+
+    Grid grid = get_grid(datahandle, -ilines_before, -xlines_before);
```

Comment:
I don't understand why origin is shifted by the before il/xl values. Is it possible to clarify this with better naming? Or a comment if not.

Reply:
I think part of my confusion stems from my interpretation of before/after as temporal. As in this is the number of inlines before running the request and this is the number after.